docs/source/recipes/lora_finetune_single_device.rst

```diff
@@ -0,0 +1,115 @@
+.. _lora_finetune_recipe_label:
+
+=============================
+LoRA Single Device finetuning
+=============================
+
+This recipe supports finetuning on next-token prediction tasks using `LoRA <https://arxiv.org/abs/2106.09685>`_,
+a technique to significantly reduce memory consumption during training whilst still maintaining competitive performance.
+
+Interested in using this recipe? Check out some of our awesome tutorials to show off how it can be used:
+
+* :ref:`Finetuning Llama2 with LoRA<lora_finetune_label>`
+* :ref:`End-to-End Workflow with torchtune<dataset_tutorial_label>`
+* :ref:`Fine-tuning Llama3 with Chat Data<chat_tutorial_label>`
+* :ref:`Meta Llama3 in torchtune<llama3_label>`
+* :ref:`Fine-Tune Your First LLM<finetune_llama_label>`
+
+We provide the following configurations out-of-the box. To learn how to customize recipes to suit your needs, check out
+our :ref:`configs tutorial <config_tutorial_label>`:
+
+.. code-block:: bash
```

Comment:
Maybe we should use this space instead to include run commands for the recipe, maybe with common overrides?